introduction.md

```diff
@@ -1,25 +1,29 @@
+The goal of the WordPress Plugin Directory is to provide a safe place for all WordPress users - from the non-technical to the developer - to download plugins that are consistent with the goals of the WordPress project.
 
-To that end, we want to ensure a simple and transparent process for developers to submit plugins for the directory. As part of our ongoing efforts to make the plugin directory inclusion process more transparent, we have created a list of developer guidelines. We strive to create a level playing field for all developers. 
+To that end, we want to ensure a simple and transparent process for developers to submit plugins for the directory. As part of our ongoing efforts to make the plugin directory inclusion process more transparent, we have created a list of developer guidelines. We strive to create a level playing field for all developers.
 
 If you have suggestions to improve the documentation, or questions about it, please email us at `plugins@wordpress.org` and let us know.
 
-## PLUGIN SUBMISSIONS
+<h2>Plugin Submissions</h2>
 
 In order to submit a plugin, [there are three steps](https://developer.wordpress.org/plugins/wordpress-org/#getting-plugins-hosted):
 
 1. Register on WordPress.org with a valid, regularly checked email address. If you are submitting a plugin on behalf of a company, use an official company email for verification.
-2. Whitelist plugins@wordpress.org in your email client to ensure you receive email communications.
+2. Whitelist `plugins@wordpress.org` in your email client to ensure you receive email communications.
 3. Submit your plugin with a brief overview of what it does and a complete, ready to go, zip of the plugin.
 
 Once a plugin is queued for review, we will review the code for any issues.  Most of the issues can be avoided by following the guidelines below. If we do find issues, we will contact the developer(s), and work towards a resolution.
 
-## DEVELOPER EXPECTATIONS
+<h2>Developer Expectations</h2>
 
-Developers, all users with commit access, and all users who officially support a plugin are expected to abide by the Directory Guidelines as well as the [Forum Guidelines](https://wordpress.org/support/guidelines/). Violations may result in plugins or plugin data (for previously approved plugins) being removed from the directory until the issues are resolved. Plugin data, such as user reviews, may not be restored, depending on the nature of the violation and the results of a peer-review of the situation. Repeat violations may result in all the author’s plugins being removed and the developer being banned from hosting plugins on WordPress.org. It is the responsibility of the plugin developer to ensure their contact information on WordPress.org is up to date and accurate, in order that they receive all notifications from the plugins team.
+Developers, all users with commit access, and all users who officially support a plugin are expected to abide by the Directory Guidelines. 
```

Comment:
Including the WordPress code of conduct?

Reply:
That doesn't officially exist yet but will be added when it does :)